### ISSUE TITLE
ComboBox: aria-labelledby is added to list container with non-existent label id, when no label prop is passed to combobox

### ISSUE THREAD

**Issue by mohitbagra** [microsoft/fluentui#18422]
### Environment Information

- **Package version(s)**: 8.16.0
- **Browser and OS versions**: all

### Describe the issue:

When no label prop is passed to ComboBox, the dropdown list container still has `aria-labelledby` attribute with a non-existent id as value. There is no check in place for whether a label prop is passed or not when setting this attribute on the list container.
I have found 2 duplicate issues which were closed due to inactivity. Creating another issue to get some eyes on this. We are getting a bunch of accessibility bugs in our product due to this.

#13525
#16536

### Please provide a reproduction of the issue in a codepen:
https://codesandbox.io/s/falling-fog-lp0cj?file=/src/App.tsx

#### Actual behavior:

aria-labelledby attribute is added to ComboBox list container even when label is not present.

#### Expected behavior:

aria-labelledby attribute should not be set when label is not present. Instead it should use the same aria-label which is passed to the Combbox.

**Comment by naksenzh** [comment]
Hi Team, @miroslavstastny , please share the ETA for this bug.

**Comment by gouttierre** [comment]
@mohitbagra - thank you for reaching out and providing the reproduction details.

@miroslavstastny - Would you be able to confirm if this is a regression, or if this behavior is an issue?
cc @smhigley

**Comment by ling1726** [comment]
Marking as high priority to get attention on this from the previous 2 older issues

Meanwhile @mohitbagra will this help you mitigate your a11y bugs: https://codesandbox.io/s/sparkling-violet-g0vsx?file=/src/App.tsx

**Comment by smhigley** [comment]
This was fixed by #18890
